### PR TITLE
Add iframe lazyloading to the core wrapper feature

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Add iframe lazyloading to the core wrapper feature

### Testing Instructions

Create an iframe wrapper via the component (example: https://www.youtube.com/embed/tgbNymZ7vqY)
Create an iframe wrapper with the module
Check the html code and make sure the loading attribute is set (loading=lazy)
Disable the lazy loading option
Check the html code and make sure the loading attribute is set (loading=eager)

### Actual result BEFORE applying this Pull Request

There is no option to set the lazyloading attribute

### Expected result AFTER applying this Pull Request

There is a option to set the lazyloading attribute

image
image

cc @bembelimen

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 530835505ed93b87d299c25cb38183dbbe257ca2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32596">issues.joomla.org/tracker/joomla-cms/32596</a>.</sub>

File: components/com_wrapper/tmpl/wrapper/default.xml [hunk 1]
```diff
@@ -90,6 +90,17 @@
...
 			</field>
+			<field
+				name="lazyloading"
+				type="radio"
+				label="COM_WRAPPER_FIELD_LAZYLOADING_LABEL"
+				default="1"
+				layout="joomla.form.field.radio.switcher"
+				validate="options"
+				>
+				<option value="0">JDISABLED</option>
```
Comment (Quy) [inline components/com_wrapper/tmpl/wrapper/default.xml]: Use Yes/No.

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 530835505ed93b87d299c25cb38183dbbe257ca2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32596">issues.joomla.org/tracker/joomla-cms/32596</a>.</sub>

**Comment by jwaisner** [comment]
@zero-24 can you check @Quy recommended change? If this is not something that needs to be retested this can be RTC.

File: components/com_wrapper/tmpl/wrapper/default.xml (hunk 1 above)
Comment (zero-24) [reply components/com_wrapper/tmpl/wrapper/default.xml]: Done

**Comment by zero-24** [comment]
> @zero-24 can you check @Quy recommended change? If this is not something that needs to be retested this can be RTC.

Done, I dont think that this does need to be retested given that i "just" changed the string to be displayed but I will leave that decision up to you.

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32596">issues.joomla.org/tracker/joomla-cms/32596</a>.</sub>

**Comment by bembelimen** [comment]
Why not using directly "lazy" and "eger" as option values? Then you can skip the ternary operators.

**Comment by richard67** [comment]
@Quy Shall I remove RTC since updates are requested?

**Comment by zero-24** [comment]
> Why not using directly "lazy" and "eger" as option values? Then you can skip the ternary operators.

Arent that values (0 + 1) bound to the CSS for the switcher?

**Comment by Quy** [comment]
> Arent that values (0 + 1) bound to the CSS for the switcher?

No. It can be any values.

File: components/com_wrapper/tmpl/wrapper/default.xml [hunk 2]
```diff
@@ -90,6 +90,17 @@
 				<option value="0">JNO</option>
 				<option value="1">JYES</option>
 			</field>
+			<field
+				name="lazyloading"
+				type="radio"
+				label="COM_WRAPPER_FIELD_LAZYLOADING_LABEL"
+				default="1"
```
Comment (Quy) [inline components/com_wrapper/tmpl/wrapper/default.xml]:
```suggestion
				default="lazy"
```

File: modules/mod_wrapper/mod_wrapper.xml [hunk 3]
```diff
@@ -101,6 +101,18 @@
 					label="MOD_WRAPPER_FIELD_TARGET_LABEL"
 					size="30"
 				/>
+
+				<field
+					name="lazyloading"
+					type="radio"
+					label="MOD_WRAPPER_FIELD_LAZYLOADING_LABEL"
+					default="1"
```
Comment (Quy) [inline modules/mod_wrapper/mod_wrapper.xml]:
```suggestion
					default="lazy"
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 4cd60a4f0443c9653a3d00788bd291d487205123<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32596">issues.joomla.org/tracker/joomla-cms/32596</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 4cd60a4f0443c9653a3d00788bd291d487205123<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32596">issues.joomla.org/tracker/joomla-cms/32596</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32596">issues.joomla.org/tracker/joomla-cms/32596</a>.</sub>

**Pull request merged**